Tutorials/Creating-a-Custom-Dashboard/index.md

```diff
@@ -233,12 +306,11 @@ Finally, we need to update the package.manifest file to load the additional cont
 }
 ```
 
-If all is setup fine we should now receive the 'Hello world' alert every time the dashboard is reloaded in the content section of Umbraco, not very helpful, yet.
+If all is setup fine we should now receive the 'Hello world' alert every time the dashboard is reloaded in the content section of Umbraco - not very helpful, at least not yet.
 
 ### Going further - Umbraco Angular Services and Directives
 
-Umbraco has a fine selection of angular directives, resources and services that you can use in your custom property editors and dashboards, the details are here: https://our.umbraco.com/apidocs/v8/ui/#/api
-
+Umbraco has a fine selection of angular directives, resources and services that you can use in your custom property editors and dashboards, the details are here: https://our.umbraco.com/documentation/Reference/API-Documentation/#backoffice-ui
 For this example it would be nice to welcome the editor by name, to achieve this we can use the **userService** here to customise our dashboard welcome message and increase friendliness:
```

Comment:
```suggestion
For this example, it would be nice to welcome the editor by name. To achieve this we can use the `userService` to customize our dashboard welcome message and increase friendliness:
```